functions.js

```diff
@@ -0,0 +1,197 @@
+/* ## Question 1
+
+Write a function that returns the largest of two numbers. If they are equal, return the second one.
+
+Example:
+
+Input: `12, 4`
+
+Output: `12`
+*/
+console.log('#1');
+
+function twoLargestNum(numbers) {
+  numbers.sort(function(a,b){
+    return b - a;
+  });
+  return numbers[0] + numbers[1]
+}
+let numbers = twoLargestNum([10,20,30,40,50])
+console.log(numbers);
+
+/*
+## Question 2
+
+Write a function named getDogAge that takes one argument: a dog's age in human years. The function returns the dog's age in dog years. The conversion rate is 1 human year to 7 dog years.
+
+Example:
+
+Input: `4`
+
+*/
+
+console.log(" ");
+console.log('#2');
+
+function getDogAge (age){
+  return age*7;
+}
+let age = getDogAge(10);
+console.log(age);
+
+/*## Question 3
+
+Write a function `arraySum` that takes in an array and returns the sum of all the elements.
+
+Example:
+
+Input: `[13,2,4,-2]`
+Output: `17`
+*/
+
+console.log(" ");
+console.log("#3");
+
+const arraySum = (arr) =>{
+  let sums = 0
+  for(let i = 0; i <= arr.length-1; i++){
+    sums+= arr[i];
+   }
+   return sums
+ }
+let result = arraySum([1,2,3,4,5])
+ console.log(result);
+
+/*
+## Question 4
+Write a function that takes in any two numbers and return their sum.
+
+Example:
+
+Input: `14, 5`
+
+Output: `19`
+*/
+console.log(" ");
+console.log("#4");
+
+function add(array){
+  let sum = 0
+  for(let i = 0; i <= array.length-1;i++){
+    sum = array[2]+array[3];
+    }
+      return sum;
+}
+let result2 = add([0,1,3,7,11])
+console.log(result2);
+
```

Comment:
This function should take in two numerical arguments returning the sum of the two.